dataflow/flex-templates/streaming_beam/README.md

```diff
@@ -0,0 +1,296 @@
+# Dataflow flex templates - Streaming Beam
```

Comment:
Can you please update the README to match the latest changes from the Java version?

https://github.com/davidcavazos/java-docs-samples/blob/dataflow-flex-templates/dataflow/flex-templates/streaming_beam_sql/README.md

Reply:
Have updated the README